daffodil-runtime1-unparser/src/main/scala/org/apache/daffodil/processors/unparsers/ElementKindUnparsers.scala

```diff
@@ -54,25 +54,22 @@ class SequenceCombinatorUnparser(ctxt: ModelGroupRuntimeData, childUnparsers: Ve
 
   override def nom = "Sequence"
 
-  // Sequences of nothing (no initiator, no terminator, nothing at all) should
-  // have been optimized away
-  Assert.invariant(childUnparsers.length > 0)
```

Comment:
In what cases can there no be zero unparsers? Seems like there's a bug elsewhere if there are zero unparsers in a sequence combinator.

Reply:
Added back in, doesn't cause any issues.  I think this was something that Mike and I threw in when we were initially debugging this issue and forgot to remove.